lib/pivo_flow/cli.rb

```diff
@@ -1,8 +1,8 @@
 module PivoFlow
   class Cli
+    include PivoFlow::State
```

Comment:
I think it doesn't give us a lot in this case - this class needs solid refactoring - we might use modules later on - current implementation is far from perfect :/

Reply:
I think the reason I initially pulled it out was that I really wanted these two lines in `pf-prepare-commit-msg` (https://github.com/dbalatero/pivo_flow/blob/f2ae3665844956b55dc05c85e254fc767aa7b8f3/bin/pf-prepare-commit-msg#L3-L4) to share code with the `pf` binary. Unfortunately since the prepare commit msg script gets copied to your `.git/hooks` directory, it's harder to simply `require 'helpful/shared/code'` from there. I briefly considered templatizing and inlining the `PivoFlow::State` module in `pf-prepare-commit-msg`, but then decided against it for this PR.

Would you like me to move it back?

Reply:
we can leave it as it is for now, but we should re-think the structure at some point!